wasm/jit/jit_amd64.go

```diff
@@ -3996,6 +3999,7 @@ func (c *amd64Compiler) compileLoad8(o *wazeroir.OperationLoad8) error {
 	moveFromMemory.To.Reg = reg
 	moveFromMemory.From.Type = obj.TYPE_MEM
 	moveFromMemory.From.Reg = reservedRegisterForMemory
+	moveFromMemory.From.Offset = -targetSizeInBytes
```

Comment:
negatives are usually odd, so you can trailing comment why this is negative? ex. `because this is accessed as memory.Buffer[ceil-targetSizeInBytes: ceil]`

same everywhere with what you decide